clients/admin-ui/src/features/plus/plus.slice.ts

```diff
@@ -563,5 +563,26 @@ export type DictSystems = {
 const EMPTY_DICT_SYSTEMS: DictSystems[] = [];
 export const selectAllDictSystems = createSelector(
   [(RootState) => RootState, plusApi.endpoints.getAllSystemVendors.select()],
-  (RootState, { data }) => data || EMPTY_DICT_SYSTEMS
+  (RootState, { data }) =>
+    data
+      ? data
+          .slice()
+          .map((ds) => {
+            const name = ds.name
+              .split(" ")
+              .map((word) =>
```

Comment:
this makes it so legal names from compass that are in all caps aren't in all caps. This fixes an UX issue where all of the all caps names go to the top when the vendors are sorted

Reply:
Small nitpick: I feel like I'd prefer for this to be in like a `toTitleCase` helper method somewhere, it's pretty hard to parse in this form and location.